src/metabase/lib/fe_util.cljc

```diff
@@ -193,14 +193,21 @@
 (defn- number->expression-arg
   [value]
   (if (u.number/bigint? value)
-    (str value)
+    (lib.options/ensure-uuid [:value {:base-type :type/BigInteger, :effective-type :type/BigInteger} (str value)])
```

Comment:
It feels like this call got duplicated in a few places. Maybe it can go into `lib.fe-util` as a function and get referenced from eg. `lib.drill-thru.quick-filter`.